Mathlib/Analysis/InnerProductSpace/Harmonic/Analytic.lean

```diff
@@ -0,0 +1,65 @@
+/-
+Copyright (c) 2025 Stefan Kebekus. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Stefan Kebekus
+-/
+import Mathlib.Analysis.Calculus.FDeriv.Symmetric
+import Mathlib.Analysis.Complex.CauchyIntegral
+import Mathlib.Analysis.Complex.Conformal
+import Mathlib.Analysis.InnerProductSpace.Harmonic.Basic
+
+/-!
+# Analyticity of Harmonic Functions
+
+If `f : ℂ → ℝ` is harmonic at `x`, we show that `∂f/∂1 - I • ∂f/∂I` is complex-analytic at `x`.
+
+TODO: As soon as PR #9598 (feat(Analysis/Complex): HasPrimitives on disc) is merged, extend this to
+show that `f` itself is locally the real part of a holomorphic function, and hence real-analytic.
+-/
+
+open Complex InnerProductSpace Topology
+
+variable
+  {f : ℂ → ℝ} {x : ℂ}
+
+/--
+If `f : ℂ → ℝ` is harmonic at `x`, then `∂f/∂1 - I • ∂f/∂I` is complex differentiable at `x`.
+-/
+theorem HarmonicAt.differentiableAt_complex_partial (hf : HarmonicAt f x) :
+    DifferentiableAt ℂ (ofRealCLM ∘ (fderiv ℝ f · 1) - I • ofRealCLM ∘ (fderiv ℝ f · I)) x := by
```

Comment:
Could you avoid `ofRealCLM`, as the fact that this is a continuous linear map is not really relevant to this statement? Using the coercion to C should be enough.

Reply:
@sgouezel Thanks for looking at the PR. I have experimented a bit, and I am unsure what a good formulation would look like. Are you thinking of one of the following?

- `DifferentiableAt ℂ ((fderiv ℝ f · 1 : ℂ → ℂ) - I • (fderiv ℝ f · I : ℂ → ℂ)) x`
- `DifferentiableAt ℂ (fun z ↦ fderiv ℝ f z 1 - I * fderiv ℝ f z I) x`

Please advise. -- Thanks.

Reply:
I was thinking of the latter, i.e., 
```
DifferentiableAt ℂ (fun z ↦ fderiv ℝ f z 1 - I * fderiv ℝ f z I) x
```